From f80f0983a21741447cc98f4ad2965288959c7268 Mon Sep 17 00:00:00 2001
From: Marc Kelly <marc.kelly@iceotope.com>
Date: Fri, 16 Mar 2018 17:07:25 +0000
Subject: [PATCH 1/1] Adding in the iceotop bmc config

---
 arch/arm/dts/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index 71cf54023d..083406a96f 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -86,6 +86,7 @@ dtb-$(CONFIG_ARCH_ZYNQ) += zynq-zc702.dtb \
 	zynq-zed.dtb \
 	zynq-zybo.dtb \
 	zynq-microzed.dtb \
+	zynq-iceotope_bmc.dtb \
 	zynq-cc108.dtb \
 	zynq-afx-nand.dtb \
 	zynq-afx-nor.dtb \
-- 
2.16.2

